presidio-anonymizer/presidio_anonymizer/engine/anonymizers/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = []
```

Comment:
why engine/anonymizers? 
what else do we have under engine?
In analyzer we have 'predefined_recognizers' directly under 'presidio_analyzer', less boilerplates, __init__.pys..

Reply:
The engine has a lot of logic in it. 
The anonymizers are the ones we can extend and the user will also be able to extend.
The anonymizers are implementing the AnonymizerAbstract. I think it is more correct to have them under engine/anonymizers and not under engine.

Reply:
ok, just know that we are chaining the structure from analyzer, also consider the option to put it in anonymizers, outside of engine

Reply:
lets talk :)

Reply:
Any reason to not user the term transfornations (as in the GO implemetation)?

Reply:
If the engine is only one module, it can be in the root folder, and anonymizers/transformations in a folder of their own:

```
presidio_anonymizer
   - anonymizers
      - fpe.py
      - replace.py
      - ...
   - anonymizer_engine.py  
   - app.py # (flask)
setup.py
```
WDYT?

Reply:
currently, the API is in the root.
Not sure the engine should be there as well.
Or we can transfer the API somewhere else although I think the root is the right place for it.

Reply:
API could either be in the package or on the root of the presidio-anonymizer folder. I'm not sure it should be an official part of the python package. WDYT?

Reply:
I like the thin structure @omri374 suggested